### PR TITLE
Update the manual section on tests.

### PR DESCRIPTION
**Pull request by bangerth**
Prompted by a recent mailing list discussion.

### CODE REVIEW

File: doc/manual/manual.tex [hunk 1]
```diff
@@ -1737,7 +1737,7 @@ \subsubsection{The Boussinesq approximation (BA)}
 satisfies a relationship of the form $\rho=\rho(T)=\rho_0(1-\alpha(T-T_0))$
 with a small thermal expansion coefficient $\alpha$ and a reference density
 $\rho_0$ that is attained at temperature $T_0$. Since the thermal expansion is
-considered small, this naturally leads to the following variant of the Boussinesq
+considered small, this naturally leads to the following variant of the the Boussinesq
```
Comment (tjhei) [inline doc/manual/manual.tex]: you seem to be undoing some useful prior changes here?!

File: doc/manual/manual.tex [hunk 2]
```diff
@@ -6417,7 +6417,7 @@ \subsubsection{Tracking finite strain}
 The plugin was tested against analytical solutions for the deformation gradient tensor in simple and pure shear as described in \url{benchmarks/finite_strain/pure_shear.prm} and \url{benchmarks/finite_strain/simple_shear.prm}.
 
 We will demonstrate its use at the example of a 2D Cartesian convection model (Figure~\ref{fig:finite_strain}):
-Heating from the bottom leads to the ascent of plumes from the boundary layer (top panel), and the
+Heating from the bottom leads to the the ascent of plumes from the boundary layer (top panel), and the
```
Comment (tjhei) [inline doc/manual/manual.tex]: same

File: doc/manual/manual.tex [hunk 3]
```diff
@@ -9703,7 +9703,7 @@ \subsection{The idea of plugins and the \texttt{SimulatorAccess} and \texttt{Int
 
 \begin{figure}[tbp]
   \centering
-  \includegraphics[width=0.95\textwidth]{plugin_graph.pdf}
+  \includegraphics[width=0.95\textwidth]{plugin_graph}
```
Comment (tjhei) [inline doc/manual/manual.tex]: same

File: doc/manual/manual.tex [hunk 4]
```diff
@@ -11464,55 +11464,104 @@ \subsubsection{Running tests}
...
     make setup_tests
-\end{verbatim}
-
-To run a set of tests, go to the build directory and execute 
-
-\begin{verbatim}
+\end{lstlisting}
+ in the build directory. To run the entire set of tests, then execute
+\begin{lstlisting}[frame=single,language=ksh]
     ctest
```
Comment (tjhei) [inline doc/manual/manual.tex]: maybe also add "-j <number_of_cores>" here?

**Comment by bangerth** [comment]
I must have accidentally saved an editor window after doing a pull. In any case, I've fixed the issues and added the `-j` description.

**Pull request merged**